deel-2-flexbox-layout/opdracht-4/styles/styles.css

```diff
@@ -1 +1,68 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+*{
+    padding:0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+.outer-col-1{
+    display: flex;
+    flex-direction: row;
+    flex-basis: 20%;
+    flex-grow:0;
+    flex-shrink:0;
+}
+
+.outer-col-2{
+    display: flex;
+    flex-direction: column;
+    flex-basis:80%;
+    flex-grow:1;
+    flex-shrink:1;
+}
+
+header{
+    flex-basis:100px;
+    flex-grow:0;
+    flex-shrink:0;
+}
+
+.inner-row{
+    display: flex;
+    flex-direction: row;
+    flex-basis: 900px;
+    flex-grow:1;
+    flex-shrink:1;
+}
+
+.inner-col{
+    display: flex;
+    flex-direction: column;
+    flex-basis: 900px;
+    flex-grow:1;
+    flex-shrink:1;
+}
+
+aside{
+    flex: 200px;
+    flex-grow:0;
+    flex-shrink:0;
+}
+
+article{
+    flex-basis: 85%;
+    flex-grow:1;
+    flex-shrink:1;
+}
+
+footer{
+    flex-basis: 15%;
+    flex-grow:0;
+    flex-shrink:0;
+}
+
```

Comment:
Ik snap niet helemaal hoe je deze hebt aangepakt. Maar het ziet goed uit. Enige is als het browser scherm minder hoog maak valt de footer er vanaf. Dat zit m denk ik in omdat je de class "inner-col" een flex-basis hebt gegeven van 900px.